lively.merger/3-way-merger.js

```diff
@@ -44,19 +66,15 @@ export function merge (o, a, b) {
         result[k] = a[k];
       } else if (a[k] !== result[k]) {
         if (a[k].isColor && result[k].isColor) {
-          const merge_result = merge(
-            JSON.parse(JSON.stringify(o[k])),
-            JSON.parse(JSON.stringify(a[k])),
-            JSON.parse(JSON.stringify(b[k]))
-          );
-          result[k] = Color.fromTuple(Object.values(merge_result));
+          if (!o[k].equals(a[k]) && !o[k].equals(b[k]) && !a[k].equals(b[k])) {
+            new MergeConflict(k, a[k], b[k]);
+            result[k] = o[k]; // TODO: should be changed later when merge conflicts can be resolved
+          } else { result[k] = a[k].equals(o[k]) ? b[k] : a[k]; }
```

Comment:
This code is exactly the same as the point code from below. Maybe we could extract a method for "special objects"?